### ISSUE TITLE
scanner failing - (zap-baseline-custom.py) missing latest api changes

### ISSUE THREAD

**Issue by hirizarry643** [ICTU/zap-baseline#1]
There has been some recent updates to the api that requires some modifications to the zap-baseline-custom.py script. Please see (https://github.com/zaproxy/zaproxy/tree/develop/build/docker)

**Comment by injcristianrojas** [comment]
I got the same issue over here. For instance, I get this error:

```python
WARNING:root:Unexpected error: <type 'exceptions.TypeError'>
Traceback (most recent call last):
  File "/zap/zap-baseline-custom.py", line 542, in main
    for url in zap.core.urls:
TypeError: 'instancemethod' object is not iterable
```

**Comment by goner00k** [comment]
I will fix this issue the coming days. What are the commandline options you are using when this error occurs?

**Comment by injcristianrojas** [comment]
I think I've fixed it in #4.

I tested it using the following:
```shell
docker build -t injcristianrojas/zap2docker-stable .
docker run -t injcristianrojas/zap2docker-stable zap-baseline-custom.py -m 2 -a -j -z "-config scanner.strength=INSANE" --active_scan -t http://172.26.0.1:8080
```
